HHS/Head-Start-TTADP
# CI: Accessibility scanner

**Description of change**
Add accessibility scan of all urls to continuous integration pipeline

Accessibility issues were found by the scan. Some of these issues are expected to be fixed by work that is already in flight, the other issues will take some investigation. I created a new issue, https://github.com/HHS/Head-Start-TTADP/issues/159, for this work and to add the `--exit` flag into the axe command. The exit flag will cause the job to fail when an accessibility issue is found.

**How to test**
- See axe accessibility scan run in [recent circle ci job](https://app.circleci.com/pipelines/github/adhocteam/Head-Start-TTADP/583/workflows/cd8b33cf-6baf-42ae-a3e0-63a2b2487fd8/jobs/2421)

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/41

**Checklist**

- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
- [x] Documentation updated - **_I simply mentioned this in the README, do you all think we need more explanation?_**
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
